stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -60,7 +66,10 @@ public data class StreamDimens(
             attachmentsContentFileUploadWidth = 250.dp,
             threadSeparatorVerticalPadding = 8.dp,
             threadSeparatorTextVerticalPadding = 2.dp,
-            messageOverlayActionItemHeight = 40.dp
+            messageOverlayActionItemHeight = 40.dp,
+            messageOptionsMaxWidth = 200.dp,
+            messageOptionsMaxHeight = 300.dp,
```

Comment:
Setting this to a smaller value to allow scroll in a popup will look weird. I think for both cases scroll should not be possible:
- The message options overlay as it is now
- Our future implementation of message options in a bottom sheet